js/Readium.js

```diff
@@ -12,10 +12,10 @@
 //  prior written permission.
 
 
-define(['readium_shared_js/globals', 'text!version.json', 'jquery', 'underscore', 'readium_shared_js/views/reader_view', 'readium_js/epub-fetch/publication_fetcher',
```

Comment:
Does this work with all AMD loaders? (RequireJS and Almond are the ones used in "multiple" and "single" bundle builds, respectively)